lib/public/views/Runs/Overview/exportRunsTriggerAndModal.js

```diff
@@ -97,15 +101,15 @@ const errorDisplay = () => h('.danger', 'Data fetching failed');
  * @return {Component} Return the view of the inputs
  */
 const exportRunsModal = (runsOverviewModel, modalHandler) => {
-    const runsRemoteData = runsOverviewModel.allRuns;
+    runsOverviewModel.loadExportableRuns();
+    const runsRemoteData = runsOverviewModel.exportableItems;
 
     return h('div#export-runs-modal', [
         h('h2', 'Export Runs'),
         runsRemoteData.match({
-            NotAsked: () => errorDisplay(),
             Loading: () => exportForm(runsOverviewModel, null, modalHandler),
             Success: (payload) => exportForm(runsOverviewModel, payload, modalHandler),
-            Failure: () => errorDisplay(),
+            Other: () => errorDisplay(),
```

Comment:
Are you sure this is a good idea? I think `NotAsked` displaying an error was a mistake, and you are hiding it here. `Other` should not lead to some error display.

Reply:
revoked, if using `NotAsked` was an error, changing it doesn't belong to this PR...